docs/cluster-management/spot-instances.md

```diff
@@ -31,7 +31,7 @@ spot_config:
   on_demand_backup: true
 ```
 
-Spot instances are not guaranteed to be available. The chances of getting spot instances can be improved by providing `instance_distribution`, a list of alternative instance types to the primary `instance_type` you specified. If left blank, Cortex will autofill `instance_distribution` with up to 2 other similar instances. Cortex defaults the `max_price` to the on-demand price of the primary instance.
+Spot instances are not guaranteed to be available. The chances of getting spot instances can be improved by providing `instance_distribution`, a list of alternative instance types to the primary `instance_type` you specified. If left blank, Cortex will autofill `instance_distribution` with the primary instance type. Cortex defaults the `max_price` to the on-demand price of the primary instance.
```

Comment:
nit: "Cortex will autofill `instance_distribution` with the primary instance type" -> "Cortex will only include the primary instance type in the `instance_distribution`"

Reply:
Yep. Doing this one too.